charts/files/user_dashboard.json

```diff
@@ -183,10 +152,14 @@
       "list": [
         {
           "allValue": "",
-          "current": {},
+          "current": {
+            "selected": false,
+            "text": "5fcc12ae-513f-4eac-a2e0-e7772f191282",
+            "value": "5fcc12ae-513f-4eac-a2e0-e7772f191282"
+          },
           "datasource": {
             "type": "prometheus",
-            "uid": "${DS_PROMETHEUS}"
+            "uid": "prometheus"
```

Comment:
doesn't this break things? looks like it wasn't exported as "share externally"?

Reply:
hmm, I'm not sure why this is being reported as a change, since this is already the contents of the file. I believe when we did 'share externally' it didn't work when @assumptionsandg was testing, but it did when we didn't do 'share externally', that's why we changed it. Jake can you advise?

Reply:
this was done in PR #21

Reply:
just merged in main, we can squash this PR